modules/datastore/src/Service/PostImport.php

```diff
@@ -116,4 +328,30 @@ public function removeJobStatus($resourceIdentifier): bool {
     }
   }
 
+  /**
+   * Remove row.
+   *
+   * @param string $resourceIdentifier
+   *   The resource identifier of the distribution.
+   */
+  public function drop($resourceIdentifier): bool {
+    try {
+      $this->datastoreService->drop($resourceIdentifier, NULL, FALSE);
+      return TRUE;
+    }
+    catch (\Exception $e) {
+      throw $e;
+    }
+  }
+
+  /**
+   * Invalidate all appropriate cache tags for this resource.
+   *
+   * @param mixed $resourceId
+   *   A resource ID.
+   */
+  public function invalidateCacheTags($resourceId): void {
```

Comment:
This is a redundant method. It's identical to DatastoreService::invalidateCacheTags, and we are already injecting DatastoreService. Remove this method, remove ReferenceLookup as a dependency, and call the function as $this->datastoreService->invalidateReferencerCacheTags().

Reply:
(need to make the DatastoreService method public, obviously)